### PR TITLE
Delete comment

### PR DESCRIPTION
**Pull request by nky4593**
Screenshot from 2019-08-05 10-25-11

Link Redmine: https://edu-redmine.sun-asterisk.vn/issues/14672

### CODE REVIEW

**Comment by nky4593** [comment]
ready

**Comment by nky4593** [comment]
ready

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -15,6 +17,18 @@ def create
...
   end
 
+  def destroy
+    if @comment.destroy
+      respond_to do |format|
+        format.html{redirect_to request.referrer || @chapter}
+        format.js
+      end
+    else
+      flash[:danger] = t ".del_fail"
```
Comment (hungct-1121) [inline app/controllers/comments_controller.rb]: failure_deleted hoặc delete_failed

Comment (nky4593) [reply app/controllers/comments_controller.rb]: done nak :v

**Comment by nky4593** [comment]
ready

**Pull request merged**